openlibrary/coverstore/code.py

```diff
@@ -61,7 +61,7 @@ def _query(category, key, value):
 
 ERROR_EMPTY = 1, "No image found"
 ERROR_INVALID_URL = 2, "Invalid URL"
-ERROR_BAD_IMAGE = 3, "Invalid Image"
+ERROR_BAD_IMAGE = 3, "save_image() failed"
```

Comment:
This is returned by the API; changing this could cause breakages in down stream code that makes POST requests to this API.

```suggestion
ERROR_BAD_IMAGE = 3, "Invalid Image"
```